internal/consensus/state.go

```diff
@@ -2660,7 +2660,7 @@ func (cs *State) calculateProposalTimestampDifferenceMetric() {
 			MessageDelay: cs.state.ConsensusParams.Synchrony.MessageDelay,
 		}
 
-		isTimely := cs.Proposal.IsTimely(cs.ProposalReceiveTime, tp, cs.state.InitialHeight)
+		isTimely := cs.Proposal.IsTimely(cs.ProposalReceiveTime, tp)
```

Comment:
We are computing that again.

We probably should move this to when we test this condition for a proposal, namely L2681.